src/borg/archiver.py

```diff
@@ -3525,8 +3530,9 @@ def define_archive_filters_group(subparser, *, sort_by=True, first_last=True):
         .. note::
 
             If you copy a repository with the lock held, the lock will be present in
-            the copy, obviously. Thus, before using borg on the copy, you need to
-            use "borg break-lock" on it.
+            the copy. Thus, before using borg on the copy from or on a different host,
+            you need to use "borg break-lock" on it.
+            (Borg is cautious and does not automatically remove stale locks made by a different host)
```

Comment:
remove parens, add dot?
or move it to before the final dot of previous sentence?